Content.Server/Voting/Managers/VoteManager.cs

```diff
@@ -550,7 +559,8 @@ public enum VoterEligibility
             All,
             Ghost, // Player needs to be a ghost
             GhostMinimumPlaytime, // Player needs to be a ghost, with a minimum playtime and deathtime as defined by votekick CCvars.
-            MinimumPlaytime //Player needs to have a minimum playtime and deathtime as defined by votekick CCvars.
+            MinimumPlaytime, //Player needs to have a minimum playtime and deathtime as defined by votekick CCvars.
+            CosmicCult // DeltaV - Player needs to be a cosmic cultist. Used by the cosmic cult gamemode.
```

Comment:
Add trailing comma so the next person doesn't also have the line above theirs :^)

```suggestion
            MinimumPlaytime, //Player needs to have a minimum playtime and deathtime as defined by votekick CCvars.
            CosmicCult, // DeltaV - Player needs to be a cosmic cultist. Used by the cosmic cult gamemode.
```